### ISSUE TITLE
Recurring crash when loading saves

### ISSUE THREAD

**Issue by byr1al** [Return-To-The-Roots/s25client#449]
Every time I try to load a savegame, the game crashes. I'm on OSX 10.11.4
The console output is this:
```
Warning: File has an old version and cannot be used (version: 25; expected: 33, file path: /Users/David/Library/Application Support/Return To The Roots/SAVE/ghgh.sav)!
```
I installed the new version which was committed yesterday and the save should thus have the same version as the current install. I tried fixing it by simply deleting the "Return To The Roots" folder in "Library/Application Support". This didn't help though, as the game is still crashing when I'm trying to load a new savegame. This time it didn't even output any errors or warnings in the terminal.

**Comment by Flamefire** [comment]
Hello and thanks for reporting. Can you give a few more information please:

1. Exact version as shown in RTTR (the alphanumeric revision)
2. How (with which version and on which pc) was this savegame created?
4. What does exactly happen? Do you get a crash report? Does the game suggest to report the crash? Does it just close? What are the last lines in the log/console?
3. Attach the crashing savegame to this issue so we can reproduce it

BTW: The warning you have seen is for a save created with an older version and not for the one you are trying to load (as the filename is shown in the message) so it is unrelated

**Comment by byr1al** [comment]
Happy to help :) This 'remake' of the game is amazing!

1. v20160410-d213668
2. The savegame was created with this version of the game and on this pc - a mac running OSX 10.11.4 (El Capitan)
3. Before I deleted the "Return To The Roots" folder in "Application Support" I got a crash report. Unfortunately I didn't save the terminal output, so I'm not sure what the last line said.
After the deletion, it just crashes without any crash report. The last line of the terminal says:
```
/Users/David/Games/s25client.app/Contents/MacOS/rttr.command: line 63: 73163 Abort trap: 6           ./bin/s25client
```

Here's the savegame file

[king.sav.zip](https://github.com/Return-To-The-Roots/s25client/files/214599/king.sav.zip)
